localstack/services/kms/provider.py

```diff
@@ -449,6 +452,41 @@ def import_key_material(
         key_obj.key_material = key_material
         return ImportKeyMaterialResponse()
 
+    def list_aliases(
+        self,
+        context: RequestContext,
+        key_id: KeyIdType = None,
+        limit: LimitType = None,
+        marker: MarkerType = None,
+    ) -> ListAliasesResponse:
+        if key_id is None:
+            return call_moto(context)
+
+        response_aliases = PaginatedList()
+
+        if kms_backends.get(context.region).keys.get(key_id) is None:
+            raise NotFoundException(f"Unable to find key '{key_id}'")
+
+        aliases_of_key = kms_backends.get(context.region).get_all_aliases().get(key_id) or []
+
+        for alias_name in aliases_of_key:
+            response_aliases.append(
+                AliasListEntry(
+                    AliasArn=f"arn:aws:kms:{context.region}:{get_account_id()}:{alias_name}",
+                    AliasName=alias_name,
+                    TargetKeyId=key_id,
+                )
+            )
+
+        page, nxt = response_aliases.get_page(
+            lambda a: a["AliasName"], next_token=marker, page_size=limit
+        )
+
+        if nxt is None:
+            return ListAliasesResponse(Aliases=page, Truncated=False)
+        else:
+            return ListAliasesResponse(Aliases=page, NextMarker=nxt, Truncated=True)
```

Comment:
nit: It can't harm to be explicit and have the `if` check here, but we could simply return line 488 directly (as `NextMarker` is an `Optional[..]` and would also be assigned `None` if no value is passed to the initializer).